packages/odyssey-design-tokens/src/palette.json

```diff
@@ -0,0 +1,32 @@
+{
+  "palette": {
+    "primary": {
+      "lighter": { "value": "{hue.blue.50.value}" },
+      "light": { "value": "{hue.blue.300.value}" },
+      "main": { "value": "{hue.blue.500.value}" },
+      "dark": { "value": "{hue.blue.900.value}" }
+    },
+    "danger": {
+      "lighter": { "value": "{hue.blue.50.value}" },
+      "light": { "value": "{hue.red.300.value}" },
+      "main": { "value": "{hue.red.500.value}" },
+      "dark": { "value": "{hue.red.900.value}" }
+    },
+    "caution": {
+      "lighter": { "value": "{hue.blue.50.value}" },
+      "light": { "value": "{hue.yellow.300.value}" },
+      "main": { "value": "{hue.yellow.500.value}" },
+      "dark": { "value": "{hue.yellow.900.value}" }
+    },
+    "success": {
+      "lighter": { "value": "{hue.blue.50.value}" },
+      "light": { "value": "{hue.green.300.value}" },
+      "main": { "value": "{hue.green.500.value}" },
+      "dark": { "value": "{hue.yellow.900.value}" }
+    },
+    "neutral": {
+      "main": { "value": "{hue.neutral.600.value}" },
+      "dark": { "value": "{hue.neutral.900.value}" }
+    }
+  }
+}
```

Comment:
Pretty sure this looks good, just want to make absolutely sure that `main` will always be 3.0:1 and below contrast ratio a11y requirement and won't be used, for example, for text anywhere.